test/units/testsuite-70.sh

```diff
@@ -268,6 +268,22 @@ systemd-cryptenroll --tpm2-public-key-pcrs=key $img_2 && { echo 'unexpected succ
 
 systemd-cryptenroll --tpm2-pcrs=key $img_2 && { echo 'unexpected success'; exit 1; }
 
+systemd-cryptenroll --tpm2-pcrs=44+8 $img_2 && { echo 'unexpected success'; exit 1; }
+
+systemd-cryptenroll --tpm2-pcrs=8 $img_2
+
+systemd-cryptenroll --tpm2-pcrs=hello $img_2 && { echo 'unexpected success'; exit 1; }
+
+systemd-cryptenroll --tpm2-pcrs=boot-loader+boot-loader-config $img_2
+
+systemd-cryptenroll --tpm2-pcrs=7+host-platform $img_2
+
+systemd-cryptenroll --tpm2-pcrs=8+boot-loader-config+11 $img_2
+
+systemd-cryptenroll --tpm2-pcrs=5+boot-loader-config+11 $img_2
+
+systemd-cryptenroll --tpm2-pcrs=5+5+11 $img_2
```

Comment:
This is going to be at least somewhat slow. I don't think there's any gain from enrolling the image a dozen times. Please move the tests for those combinations to the unit tests in `test-parse-util`, and actually check that the parse result is correct.

Reply:
> This is going to be at least somewhat slow. I don't think there's any gain from enrolling the image a dozen times. Please move the tests for those combinations to the unit tests in `test-parse-util`, and actually check that the parse result is correct.

Alright sir. I'll add the unit tests for tpm2_pcr_mask_from_string() function.

Reply:
That's fine. We don't need a custom message.

Reply:
Awesome. Thank you sir.